core/trie2/triedb/pathdb/disklayer.go

```diff
@@ -0,0 +1,130 @@
+package pathdb
+
+import (
+	"sync"
+
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/core/trie2/trieutils"
+)
+
+var _ layer = (*diskLayer)(nil)
+
+// Allows access to the underlying database.
+// Nodes are buffered in memory and when the buffer size reaches a certain threshold,
+// the nodes are flushed to the database.
+type diskLayer struct {
+	root    felt.Felt // The corresponding state commitment
+	id      uint64
+	db      *Database
+	cleans  *cleanCache // Clean nodes that are already persisted in the db
```

Comment:
Is `persisted` the right word, I think it has to be with the name of the method `persist` but to be honest is not very descriptive from my PoV. What does a persisted node means, that it has already been written to disk?